proposals/type-imports/Overview.md

```diff
@@ -51,8 +51,10 @@ There are several requirements for opaque type definitions (called "private" in
 * In a similar vein, encapsulation must be forward-compatible with the addition of explicit casts. It ought to be possible to inject an encapsulated value into `anyref` and cast it back, in order for private types to participate in the same anyref-based type escape hatches as other references, and to avoid more complicated type system machinery. That is, it must be possible to compare with or cast _to_ a private type, but not _through_ it.
 
 Together, these requirements and goals necessitate that (1) private types are nominal, in order to distinguish them from one another, (2) values of private type share a representation with other reference types, in order to make private types suitable for imports, and (3) these values have a distinguished representation that maintains enough type information to distinguish them from values of other type.
-The latter in turn necessitates that such values are allocated -- as is typically the case for external, host-implemented references as well.
-However, the design presented here does not enable the formation of cycles, so that simple reference counting techniques are applicable and no GC is required (though possible).
+The latter in turn necessitates that such values are allocated -- as is typically the case for external, host-implemented references as well
+(once Wasm has other forms of allocation, such as for structs in the [GC proposal](https://github.com/WebAssembly/gc), they can be used for this purpose, avoiding extra levels of exposing, see [below](#forward-compatibility-with-gc-proposal)).
```

Comment:
```suggestion
(once Wasm has other forms of allocation, such as for structs in the [GC proposal](https://github.com/WebAssembly/gc), they can be used for this purpose, avoiding extra levels of indirection, see [below](#forward-compatibility-with-gc-proposal)).
```
I don't grok "exposing"; maybe this is what you mean?

Reply:
Yeah, no idea how that got in there.